src/worker.js

```diff
@@ -10,41 +10,26 @@ import { isConfigAtHomeRoot } from './is-config-at-home-root'
 
 process.title = 'linter-eslint helper'
 
-const ignoredMessages = [
-  // V1
-  'File ignored because of your .eslintignore file. Use --no-ignore to override.',
-  // V2
-  'File ignored because of a matching ignore pattern. Use --no-ignore to override.',
-  // V2.11.1
-  'File ignored because of a matching ignore pattern. Use "--no-ignore" to override.',
-  // supress warning that the current file is ignored by eslint by default
-  'File ignored by default.  Use a negated ignore pattern (like "--ignore-pattern \'!<relative'
-    + '/path/to/filename>\'") to override.',
-  'File ignored by default. Use "--ignore-pattern \'!node_modules/*\'" to override.',
-  'File ignored by default. Use "--ignore-pattern \'!bower_components/*\'" to override.',
-]
+function lintJob({ cliEngineOptions, contents, eslint, filePath }) {
+  const cliEngine = new eslint.CLIEngine(cliEngineOptions)
```

Comment:
That seems like it could be a good idea. It looks like  `CLIEngine` constructor checks the `--rulesdir` directory for updates only once when it's constructed, so this would mean that any changes to the rules in the rules directory wouldn't be reflected immediately. (I suspect it's pretty rare for users to modify the rules directory in the common case.)

Also, it's worth noting that the `cli` API that we previously used constructs a new instance of `CLIEngine` on every invocation too, so I doubt this will hurt performance compared to the previous behavior (although there might be room for improvement).